plasmapy/particles/particle_collections.py

```diff
@@ -6,9 +6,9 @@
 import collections
 import numpy as np
 
-from typing import *
+from typing import *  # noqa: F403
```

Comment:
```suggestion
from typing import Callable, List, Iterable, Union
```

and if I missed any, a test failure will tell us about it 😄